integration-libs/cds/src/profiletag/services/profiletag-event.service.spec.ts

```diff
@@ -107,6 +108,34 @@ describe('ProfileTagEventTracker', () => {
     expect(nativeWindow.Y_TRACKING.eventLayer.push).not.toHaveBeenCalled();
   });
 
+  it(`Should not add config parameters to the q array if PT is already loaded`, () => {
+    const profileTagLoaded$ = profileTagEventTracker.addTracker();
+    let subscription = profileTagLoaded$.subscribe();
+    const baseSite = 'electronics-test';
+    getActiveBehavior.next(baseSite);
+    subscription.unsubscribe();
+
+    expect(nativeWindow.Y_TRACKING.q[0][0]).toEqual({
+      ...mockCDSConfig.cds.profileTag,
+      tenant: mockCDSConfig.cds.tenant,
+      siteId: baseSite,
+      spa: true,
+    });
+    expect(nativeWindow.Y_TRACKING.eventLayer.push).not.toHaveBeenCalled();
+
+    // reset the mock correctly so that the existing script is detected
+    spyOn(mockedWindowRef.document, 'querySelector').and.returnValue(
+      {} as Element
+    );
+    // retrigger profile-tag
+    subscription = profileTagLoaded$.subscribe();
+    getActiveBehavior.next(baseSite);
```

Comment:
No need to tell what base site it currently is as it was defined already from line 115

Please remove

Reply:
done.